pkg/namesgenerator/names-generator.go

```diff
@@ -136,6 +136,9 @@ var (
 		// Stefan Banach - Polish mathematician, was one of the founders of modern functional analysis. https://en.wikipedia.org/wiki/Stefan_Banach
 		"banach",
 
+		// Buckaroo Banzai and his mentor Dr. Hikita perfectd the "oscillation overthruster", a device that allows one to pass through solid matter. - https://en.wikipedia.org/wiki/The_Adventures_of_Buckaroo_Banzai_Across_the_8th_Dimension
```

Comment:
Iff we can make BTTF references (line 193), why not BB? :)

Reply:
My diff adds Vera Rubin's entry at line 651 but it doesn't show up in the pull request diff. Strange.